conference.js

```diff
@@ -1938,8 +2025,37 @@ export default {
             .then(() => {
                 mediaDeviceHelper.setCurrentMediaDevices(devices);
                 APP.UI.onAvailableDevicesChanged(devices);
+                this.updateVideoIconEnabled();
             });
     },
+    /**
+     * Determines whether or not the video button should be enabled.
+     */
+    updateVideoIconEnabled() {
+        const videoMediaDevices
+            = mediaDeviceHelper.getCurrentMediaDevices().videoinput;
+        const hasPermissions
+            = videoMediaDevices !== undefined
+                && videoMediaDevices.filter(d => d.label !== '').length > 0;
```

Comment:
This won't work with temasys as temasys has labels even if permission is denied, but I also can't get into a flow where this would cause a problem.